src/frontend/components/Offscreen.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+/**
+ * Visually move an element offscreen, so it is not visible while still present in the
+ * accessibility tree and for screen reader users.
+ */
+export const Offscreen = styled.div`
+  position: absolute;
+  left: -9999rem;
```

Comment:
This is the best way I think to hide thinks for accessiblity purpose.
Screen readers can access to the element but we are sure that the element will not be rendered visually.

```suggestion
  border: 0;
  clip: rect(0,0,0,0);
  clip-path: inset(50%);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
```

Reply:
Ok, I never had trouble with the technique I used but if this is 100% to work all the time I'll change it :)